src/terra-dev-site/guide/Internationalization/AddingTranslationsForALocaleNotSupportedByTerraUI.d.guide.md

```diff
@@ -0,0 +1,58 @@
+# Adding Translations For A Locale Not Supported By Terra UI
+
+Terra UI provides translations for [the following locales](https://github.com/cerner/terra-toolkit/blob/master/scripts/aggregate-translations/i18nSupportedLocales.js) for any string used in our components. In some cases, you may need to support locales outside of the ones we provide by default.
+
+As an example, let's say you need to support Polish (pl) in your app.
+
+In the aggregate translation functions, set the locales option and include the `pl` locale.
+
+If you only need to support English and Polish, you can call `aggregateTranslations` function with the following:
+
+```js
+const aggregateTranslations = require ('terra-toolkit/scripts/aggregate-translations/aggregate-translations');
+
+aggregateTranslations({ locales: ['en', 'pl']});
+```
+
+If you want to support all of the Terra UI supported locales in addition to Polish, you can call `aggregateTranslations` fuction with the following:
+
+```js
+const aggregateTranslations = require ('terra-toolkit/scripts/aggregate-translations/aggregate-translations');
+const supportedLocales = require('terra-toolkit/scripts/aggregate-translations/i18nSupportedLocales');
+
+
+aggregateTranslations({ locales: [...supportedLocales, 'pl']});
+```
+
+Similarly, [you can add a `terraI18n.config.js` file to the root of your project with options to pass to aggreaget-translations when its called](https://github.com/cerner/terra-toolkit/blob/master/docs/AggregateTranslations.md#terrai18nconfig-example).
```

Comment:
pass to *aggregate-translations

Reply:
Updated: https://github.com/cerner/terra-ui/pull/132/commits/576255a658865bd7bfbc7e454abbbce66e51b2d2